plugin/rpcplugin/supervisor_test.go

```diff
@@ -104,6 +119,10 @@ func TestSupervisor_PluginCrash(t *testing.T) {
 			return nil
 		}
 
+		func (p *MyPlugin) OnDeactivate() error {
+			return nil
+		}
```

Comment:
This test actually wasn't functioning properly at all. Since this method wasn't defined in the plugin, the RPC on line 141 was optimized out entirely, so it always returned with no error, even if the process wasn't actually running. Which is a great optimization... but not what we want for this test.